README.md

```diff
@@ -48,9 +48,9 @@ Here's an overview of the commands that are supported by this library. See the f
 - Uncomment [this define](https://github.com/maland16/daly-bms-uart/blob/main/daly-bms-uart.h#L8) to get lots of debug printing to the Arduino Serial Monitor. I added these statements to help as I developed the code and ran into issues. Beyond that, I've done my best to comment extensively.  
 
 ## Future Improvements
+### Clean up/corrections  
+First & foremost there are a lot of wacky things in this repo that work, but are not done the best possible way. Cleaning things up and correcting bad coding practices would be huge.   
```

Comment:
sorry about that, my coding skills are not top level, when you have the time, show me the right way and i will do the work

Reply:
There are wacky things done here by both of us haha, I didn't mean your changes specifically